patches.js

```diff
@@ -16,9 +18,10 @@ const getLocalEndpoint = () => {
   return process.env.LOCALSTACK_ENDPOINT || `https://${host}:${port}`;
 };
 
-const patchConfigManagerLoader =  () => {
+// Patchs the awscloudformation provider plugin to deploy the resources into LocalStack
+const patchConfigManagerLoader = () => {
   try {
-    const configManagerPath = snapshot_path+'amplify-provider-awscloudformation/lib/configuration-manager';
+    const configManagerPath = `${snapshot_path}amplify-provider-awscloudformation/lib/configuration-manager`;
```

Comment:
nit: could convert `snapshot_path` (snake_case) -> `snapshotPath` (camelCase), to be consistent with variable naming.